src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ChartControl.cs

```diff
@@ -4,40 +4,40 @@
 using System.Drawing;
 using System.Windows.Forms.DataVisualization.Charting;
 
-namespace WinformsControlsTest
+namespace WinFormsControlsTest;
+
+[DesignerCategory("default")]
```

Comment:
```suggestion
[DesignerCategory("Default")]
```

Can you do a quick check, if we use the captial 'D'?